### PR TITLE
add: `flatpak-git`

### PR DESCRIPTION
**Pull request by WRM-42**

### CODE REVIEW

File: packages/flatpak-git/flatpak-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name="flatpak-git"
+pkgname="flatpak"
+repology=("project: ${pkgname}")
```
Comment (wizard-28) [inline packages/flatpak-git/flatpak-git.pacscript:3]:
```suggestion

```

Not needed for `-git` pacscripts.

File: packages/flatpak-git/flatpak-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  ./autogen.sh
+}
+
+build() {
+  ./configure
+  make -j"$(nproc)"
+}
+
+install() {
+  sudo make install DESTDIR="/usr/src/pacstall/${pkgname}"
```
Comment (wizard-28) [inline packages/flatpak-git/flatpak-git.pacscript:27]:
```suggestion
  sudo make install DESTDIR="${STOWDIR}/${pkgname}"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by wizard-28** [comment]
The CI fails to install the package. See the logs and fix the pacscript.

**Comment by WRM-42** [comment]
Blocked by #954

**Pull request closed without merging**